x-pack/plugins/ml/public/file_datavisualizer/components/import_settings/simple.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React from 'react';
+
+import {
+  EuiFieldText,
+  EuiFormRow,
+  EuiCheckbox,
+} from '@elastic/eui';
+
+export function SimpleSettings({
+  index,
+  initialized,
+  onIndexChange,
+  createIndexPattern,
+  onCreateIndexPatternChange,
+  indexNameError,
+}) {
+  return (
+    <React.Fragment>
+      <EuiFormRow
+        label="Index name"
+        isInvalid={indexNameError !== ''}
```

Comment:
Just a clarifying question - for the `isInvalid` prop - seeing this specific check for empty string a lot. Does this mean `indexNameError` is getting initially defaulted to an empty string somewhere?

Reply:
yes, in the default state of the `ImportView` component.